khmer/khmer_args.py

```diff
@@ -356,25 +361,29 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0,
+                        sysmem=psutil.virtual_memory().available):
```

Comment:
I'd suggest leaving this as something default ('None'?) and then calculating it in body of the function if needed; then you can pass it in for the tests w/o making the default def that ugly.

Also, if it's an internal thing that will only be used by tests, suggest naming it _sysmem or something else.